### ISSUE TITLE
Fix nav bar at the top

### ISSUE THREAD

**Issue by ojokne** [ojokne/quiz#2]
The nav bar should be fixed when the user scrolls up for easy accessibility of the menu

**Comment by ojokne** [comment]
The margins have been distorted
